app/src/main/res/layout/fragment_change_password.xml

```diff
@@ -0,0 +1,148 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:bind="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+        <variable
+            name="viewModel"
+            type="com.framgia.englishconversation.screen.changePassword.ChangePasswordViewModel"/>
+
+    </data>
+
+    <android.support.constraint.ConstraintLayout
+        android:foregroundGravity="center"
+        android:paddingBottom="@dimen/dp_16"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content">
+
+        <TextView
+            android:paddingRight="@dimen/dp_120"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingRight` you should probably also define `paddingLeft` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.